src/ProjectNeon/Assets/Features/DeckBuilder/Scripts/DeckBuilderState.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/**
```

Comment:
Don't include redundant comments. 

If there are any comments, then should only be explaining unique implementations.

Reply:
@SilasReinagel Ok, I'll remove them